### PR TITLE
UserGuide.adoc: Update Add command

### LINKED ISSUE #107: Update Add feature in User Guide.
Update Add feature in User Guide.

### PR DESCRIPTION
**Pull request by Amoscheong97**
Update UG for add command

### CODE REVIEW

**Review by khsc96 — CHANGES_REQUESTED** [review]
Nice work, but i think we are standardising on having sub points for "Description", "Format" and "Examples"

**Review by khsc96 — APPROVED** [review]
LGTM @JeremyLoh @Exeexe93 @Cherweijie I think we standardise with this format as shown in Amos's add command.
Bold fields marks compulsory fields, perhaps we can also use [TIP} to indicate any information given in format.

**Review by JeremyLoh — DISMISSED** [review]
Looks good to merge.

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -52,13 +52,14 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
 
 ==== Description
 * Adds a new order to the list of orders
 
 ==== Format
-* `add` `tid/*TRANSACTION_ID*` `n/*NAME*` `a/*ADDRESS*` `p/*PHONE_NUMBER*` `*ts/DELIVERY_DATE_&_TIME*` `w/*WAREHOUSE_LOCATION*` `pm/*PAYMENT_METHOD*` `[c/COMMENTS_BY_CUSTOMER]` `[type/TYPE_OF_ITEM]`
+* `add` `tid/TRANSACTION_ID` `n/NAME` `a/ADDRESS` `p/PHONE_NUMBER` `ts/DELIVERY_DATE_&_TIME` `w/WAREHOUSE_LOCATION` `pm/PAYMENT_METHOD` `[c/COMMENTS_BY_CUSTOMER]` `[type/TYPE_OF_ITEM]`
 
 [TIP]
 All the fields (in bold) are compulsory except for the fields [c/COMMENTS_BY_CUSTOMER] and [type/TYPE_OF_ITEM]
```
Comment (JeremyLoh) [inline docs/UserGuide.adoc]: This is outdated. Maybe can say like the order of prefix fields (e.g. n/NAME p/PHONE_NUMBER) does not need to be in a specific order? (Can interchange fields)

Comment (khsc96) [reply docs/UserGuide.adoc]: if not also can just don't include first perhaps we can revisit and see if there's any important notes or tips for user to take note of.

Comment (JeremyLoh) [reply docs/UserGuide.adoc]: Yes, I think you can just remove for now. Can always add back later.

**Review by khsc96 — CHANGES_REQUESTED** [review]
Change according to the review jeremy gave, thanks!

**Review by khsc96 — APPROVED** [review]
LGTM

**Review by JeremyLoh — APPROVED** [review]
LGTM

**Pull request merged**